### PR TITLE
how-to/scale: add a note in with-ansible document

### PR DESCRIPTION
**Pull request by elvizlai**
### What is changed, added or deleted? 

This PR changes a sentence into a note in the `with-ansible.md`.

### What is the related PR or file link(s)? 
N/A

### Which version does your change affect? 

dev, v2.1, v3.0

### CODE REVIEW

File: v3.0/how-to/scale/with-ansible.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ aliases: ['/docs-cn/op-guide/ansible-deployment-scale/']
 
 # 使用 TiDB Ansible 扩容缩容 TiDB 集群
 
-TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。以下缩容示例中，被移除的节点没有混合部署其他服务；如果混合部署了其他服务，不能按如下操作。
+TiDB 集群可以在不影响线上服务的情况下进行扩容和缩容。
+
+> **Warning:**
```
Comment (TomShawn) [inline v3.0/how-to/scale/with-ansible.md]:
```suggestion
> **注意：**
```

**Comment by elvizlai** [comment]
> Rest LGTM.
> @elvizlai Thanks for your contribution. Does your change also apply to v2.1 or dev version?

No, only for v3.0 doc.
Should I also make similar note for v2.1 or dev in this pr?

**Comment by TomShawn** [comment]
> > Rest LGTM.
> > @elvizlai Thanks for your contribution. Does your change also apply to v2.1 or dev version?
> 
> No, only for v3.0 doc.
> Should I also make similar note for v2.1 or dev in this pr?

Yes. Please address the comment above and make the change to v2.1 and dev in this PR.

**Comment by elvizlai** [comment]
@TomShawn PTAL

**Pull request merged**